festim/boundary_conditions/fluxes/surface_kinetics.py

```diff
@@ -0,0 +1,184 @@
+from festim import FluxBC
+from fenics import *
+import sympy as sp
+
+
+class SurfaceKinetics(FluxBC):
+    """
+    FluxBC subclass allowing to include surface processes in 1D H transport simulations:
+
+    .. math::
+
+        \dfrac{d c_{\mathrm{s}}}{dt} = J_{\mathrm{bs}} - J_{\mathrm{sb}} + J_{\mathrm{vs}};
+
+    .. math::
+
+        -D \dfrac{\partial c_{\mathrm{m}}}{\partial x} = -\lambda_{\mathrm{IS}} \dfrac{\partial c_{\mathrm{m}}}{\partial t}
```

Comment:
This wouldn't be true for the right hand side boundary. Let's use the generic nabla operator instead. What do you think?

```suggestion
        -D \nabla c_\mathrm{m} \cdot \mathbf{n} = -\lambda_{\mathrm{IS}} \dfrac{\partial c_{\mathrm{m}}}{\partial t}
```

Reply:
\nabla command doesn't work

Reply:
oh really? hm that's weird cause it works here: $\nabla$

Reply:
Oh yes because python interprets \n as a line break. Adding `r"""` before the docstrings sign fixes it:

![image](https://github.com/festim-dev/FESTIM/assets/40028739/258526be-d2f9-4b9f-b0f7-01c43ac9f1aa)